aerogear/offix-android
# feat: Offline Support with nested schema

### Description
Inside the `interceptAsync()` of `ConflictInterceptor` class, I have checked for the network connection. If the network connection is not there, and if the request operation is a mutation, I store them in an array list as of now. 
Whenever the net comes, we wait for a mutation request from the user side. Upon getting it, we fetch the mutation requests stored in the list and replicate them back to the server. 
We are not providing UI bindings to the user currently.

### How to test it?
- Start server.
- Go offline. 
- Make some mutations. (For updating a task, click on the task and provide the details in it. )
- Come online.
- Make any mutation. And you will see the results of the mutations done when offline along with the result of the mutation done when the network connection is regained.

**Also trying to study request so that we can parse it accordingly and store important things in database and get rid of array list**
